docs/getting-started/quick-start.md

```diff
@@ -4,12 +4,247 @@ sidebar_position: 2
 
 # Quick Start
 
+_Service Binding Operator should be [installed](./installing-service-binding.md) on the cluster prior to running this quick start guide._
+
 In this section we are providing a sample application that one can deploy and just use to play around.
 
-## K8S
+## Kubernetes
+
+### Application
+
+An application is a process running within a container. Examples include a NodeJS Express application, a Ruby Rails application or a Spring Boot application.
```

Comment:
What is the aim of this paragraph? The statement is rather vague, and can create more questions than close some. Quick start should focus on a golden path and describe just single scenario that really works.